decentraland/components/presentational/__snapshots__/LinkToEtherscan.test.js.snap

```diff
@@ -0,0 +1,14 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`LinkToEtherscan renders the component null account 1`] = `""`;
```

Comment:
Does a component that uses this component this mind `""` being the result in the `null` case?

Reply:
The button isn't be showed if the user has no account created, will be better if we have a disabled icon or show an alert to the user?

Reply:
Hmm, I'll have to think about that. Depends what screen this will be on.

For the purposes of this thread, I just wanted to make sure the wrapping render function didn't mind the `""`, and it sounds like it behaves as I would have expected given the choice to show `""`.

Reply:
Oh, got it. The `render()` functions is returning `null`

Reply:
Feel free to resolve this

Reply:
Resolving this.